src/course-updates/update-form/UpdateForm.jsx

```diff
@@ -73,7 +72,7 @@ const UpdateForm = ({
                   <DatePicker
                     name="date"
```

Comment:
maybe it would be better to add validation to schema?

Reply:
Done